src/execute/julia.ts

```diff
@@ -109,7 +118,13 @@ export const juliaEngine: ExecutionEngine = {
   },
 
   markdownForFile(file: string): Promise<MappedString> {
-    return Promise.resolve(mappedStringFromFile(file));
+    if (isJupyterPercentScript(file)) {
```

Comment:
Should'nt we use `isJuliaPercentScript()` ? Probably equivalent, but would maybe be clearer for future us 🤔 

Or am I missing something ?

Reply:
no I think you're right, this was copied over from the jupyter implementation and might as well use `isJuliaPercentScript`